packages/frontend/app/services/current-user.ts

```diff
@@ -68,30 +80,39 @@ export default class CurrentUserService extends Service {
     return toHex(this.publicKey);
   }
 
+  get signingKeyHex(): string {
+    if(!this.publicSigningKey) return '';
+
+    return toHex(this.publicSigningKey);
+  }
+
   get shareUrl(): string {
-    const uri = `${ENV.host}/invite?name=${this.name}&publicKey=${this.uid}`;
+    const uri = `${ENV.host}/invite?name=${this.name}&publicKey=${this.uid}&publicSigningKey=${this.signingKeyHex}`;
```

Comment:
the signing key should not be a part of the share URL. it's already long enough as is
since signing isn't required for direct communication, there should be some other means of passing it to your friends -- especially as signing keys may change. Maybe the public signing key is passed along with each message, since if a  message is signed, the public key is required for verification.

Reply:
Now that we think about it, users never need each others public signing keys outside of votes, which will store them already, so we can remove the stuff to save them for other users altogether.